omnibot/services/slack/bot.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, team, name, data):
         self._interactive_component_handlers = []
         self._slash_command_handlers = []
         self._message_handlers = []
+        self._reaction_handlers = []
```

Comment:
each of these come in via different slack webhook -- reaction handler is sharing with message handler and will the only one with two. Is this the desired architecture?